### DISCUSSION TITLE
posts/how-to-create-a-training-in-kypo/

### DISCUSSION THREAD

**Discussion by giscus** [brant-ruan/fernweh-comments#19] [Announcements]
# posts/how-to-create-a-training-in-kypo/

How to Create a Training in KYPO

https://blog.wohin.me/posts/how-to-create-a-training-in-kypo/

**Comment by mouazma798** [comment]
Repected, 
                I am writing to request you help in the deployment of KYPO on openstack platform. I have installed openstack (Kolla-ansible) on my server and did KYPO pre-configuration on Openstack platform. Created instances, public/private network, router. I am able SSH the openstack instances from the host server but the instances can't ping 8.8.8.8. It means instances do not have access to internet. Could you help me in resolving this matter. Moreover, I have tried to install KYPO terraform on devstack. After the successful installation of OpenStack Devstack and doing some pre-configuration for the installation of KYPO, I got error i.e. images can't be uploaded on openstack server. I would be very glad if you help me out in solving this issue

**Reply by brant-ruan** [reply]
Sorry that I have almost forgotten the deployment details of KYPO. I suggest you propose issues to the developing team.
